include/zenkit-capi/CutsceneLibrary.h

```diff
@@ -1,6 +1,7 @@
 // Copyright © 2023. GothicKit Contributors
 // SPDX-License-Identifier: MIT
 #pragma once
+#include "Archive.h"
```

Comment:
The file is missing, did you forget to commit it?

Reply:
Yep, indeed, fixed